ui/lib/core/addon/utils/client-count-utils.js

```diff
@@ -4,7 +4,36 @@
  */
 
 import { parseAPITimestamp } from 'core/utils/date-formatters';
-import { compareAsc, getUnixTime } from 'date-fns';
+import { compareAsc, getUnixTime, isAfter, isBefore } from 'date-fns';
+
+// returns array of VersionHistoryModels for noteworthy upgrades (1.9, 1.10, 1.16)
+// that occurred during the queried activity data
+export const filterVersionHistory = (versionHistory, activity) => {
```

Comment:
Would love to see one more test of this method (would also help to communicate the formats of version history and activity!)

Reply:
Oops! I have a test, just forgot to push!